content/en/users/_index.md

```diff
@@ -12,41 +12,31 @@ menu:
 
 You can ask for more information about the services at: [here](https://www.egi.eu/more-information/)
 
-## Ordering EGI services
+## Accessing EGI services
 
-You can order EGI services from:
+You can access EGI services from:
 
-* [EGI marketplace](https://marketplace.egi.eu/), or
-* [EOSC marketplace](https://marketplace.eosc-portal.eu/services?providers=1)
+* https://www.egi.eu/services/ 
 
-### Payments Model
+Depending on the access conditions, a service (or an instance of the service) may be open for any user, OR it may require requesting access (Ordering). The EGI Website together with the connected EGI Marketplace streamlines the process.  
```

Comment:
```suggestion
Depending on the access conditions, a service (or an instance of the service) may be open
for any user, OR it may require requesting access (Ordering). The EGI Website together with
the connected EGI Marketplace streamlines the process.  
```